### PR TITLE
AEA-1464 Feature/v1.0.0 final

### PR DESCRIPTION
**Pull request by DavidMinarsch**
## Proposed changes

This PR:
- adds history item
- adds fixes for simple oracle tests
- adds latest oracle contracts
- parses for JSONDecodeError in cosmos/fetchai plugins
- fixes a CVE
- removes `sys.exit` where not needed
- misc other fixes

## Fixes

-

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

-

### CODE REVIEW

File: plugins/aea-ledger-cosmos/aea_ledger_cosmos/cosmos.py [hunk 1]
```diff
@@ -972,7 +975,11 @@ def _try_execute_wasm_query(
             json.dumps(query_msg),
         ]
 
-        return json.loads(self._execute_shell_command(command))
+        cli_stdout = self._execute_shell_command(command)
```
Comment (solarw) [inline plugins/aea-ledger-cosmos/aea_ledger_cosmos/cosmos.py:978]: just create a helper method for this pattern?

```
execute
try:
 load_json
 ...
 ```

**Review by solarw — APPROVED** [review]
LGTM

File: plugins/aea-ledger-cosmos/aea_ledger_cosmos/cosmos.py (hunk 1 above)
Comment (DavidMinarsch) [reply plugins/aea-ledger-cosmos/aea_ledger_cosmos/cosmos.py:978]: Yeah sure, today I care about speed though :D

**Review by jrriehl — APPROVED** [review]
LGTM

**Pull request merged**